Assignment1/Assignment1/MovieData.swift

```diff
@@ -31,19 +31,17 @@ extension String {
 }
 
 class MovieData: NSObject {
-    var titles: [String] = []
+    var episodes: Array<Episodes>?
```

Comment:
GOod that it is an optional incase the array is NIL so we are forced to check it